s4e-web/src/app/views/map-view/map/map.component.ts

```diff
@@ -52,6 +88,21 @@ export class MapComponent implements OnInit, OnDestroy {
     }
 
     this.activeProduct$.pipe(untilDestroyed(this)).subscribe(gr => this.updateLayers(gr));
+
+    const markerStyle = new Style({
+      image: new Icon(/** @type {olx.style.IconOptions} */ ({
+        anchor: [0.5, 220],
+        anchorXUnits: 'fraction',
+        anchorYUnits: 'pixel',
+        opacity: 0.75,
+        scale: 0.25,
+        src: 'https://cdn2.iconfinder.com/data/icons/ui-26/128/map-pin-location-256.png'
+      }))
+    });
+    this.map.getLayers().push(new VectorLayer({
+      source: this.markerSource,
+      style: markerStyle,
+    }));
   }
 
   ngOnDestroy(): void {
```

Comment:
You should call `this.selectedLocationSub.unsubscribe();` in `ngOnDestroy()` as well as nullify `this.selectedLocationSub`